src/app/[language]/password-change/page-content.tsx

```diff
@@ -118,13 +119,15 @@ function Form() {
                 name="password"
                 label={t("password-change:inputs.password.label")}
                 type="password"
+                testId="password"
               />
             </Grid>
             <Grid item xs={12}>
               <FormTextInput<PasswordChangeFormData>
                 name="passwordConfirmation"
                 label={t("password-change:inputs.passwordConfirmation.label")}
                 type="password"
+                testId="passwordConfirmation"
```

Comment:
Let's use in `textId` [kebab case ](https://www.theserverside.com/definition/Kebab-case)

like:

```
testId="password-confirmation"
```

Please also check all files :)

Reply:
Done